### PR TITLE
HDDS-3031. OM HA- Client requests get LeaderNotReadyException after OM's restart.

### PR DESCRIPTION
**Pull request by bharatviswa504**
## What changes were proposed in this pull request?

Fix LeaderNotReady exception which is thrown for write requests after OM restarts.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-3031

## How was this patch tested?

Deployed on the cluster and tried the scenario. With the PR fix, now able to execute client requests after OM restart.

I will see if I can add UT for this.

### CODE REVIEW

File: hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeCreateRequest.java [hunk 1]
```diff
@@ -164,7 +164,7 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
           // Replay implies the response has already been returned to
           // the client. So take no further action and return a dummy
           // OMClientResponse.
-          LOG.debug("Replayed Transaction {} ignored. Request: {}",
+          LOG.info("Replayed Transaction {} ignored. Request: {}",
```
Comment (hanishakoneru) [inline hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeCreateRequest.java]: Keeping this log message at info level will flood the log file. It prints the Request body also.

File: hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/s3/multipart/S3InitiateMultipartUploadRequest.java [hunk 2]
```diff
@@ -203,11 +203,7 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
       omClientResponse = new S3InitiateMultipartUploadResponse(
           createErrorOMResponse(omResponse, exception));
     } finally {
-      if (omClientResponse != null) {
-        omClientResponse.setFlushFuture(
-            ozoneManagerDoubleBufferHelper.add(omClientResponse,
-                transactionLogIndex));
-      }
+
```
Comment (hanishakoneru) [inline hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/s3/multipart/S3InitiateMultipartUploadRequest.java]: Missed adding addResponseToDoubleBuffer() here.

**Comment by bharatviswa504** [comment]
Thank You @hanishakoneru for the review.
Addressed review comments.

**Comment by hanishakoneru** [comment]
LGTM. +1.
Thanks for fixing this. Will merge this PR.

**Pull request merged**